support/Pipelines/Bbh/InitialData.py

```diff
@@ -141,6 +149,7 @@ def generate_id(
 
     Scheduling options:
       id_input_file_template: Input file template where parameters are inserted.
+      control: Set to True to control BBH physical parameters (default: False).
```

Comment:
Please add some more docs to explain this option, maybe in the documentation text above. Note that with this option set to False the horizon masses and spins in the generated data will differ from the input parameters. And with this option set to True a postprocessing control loop will adjust the input parameters to drive the horizon masses and spins to the specified values.